pages.it/common/node.md

```diff
@@ -0,0 +1,24 @@
+# node
+
+> Piattaforma JavaScript Server-side (Node.js).
+> Maggiori informazioni: <https://nodejs.org>.
+
+- Esegue un file JavaScript:
+
+`node {{percorso/del/file}}`
+
+- Avvia una REPL (shell interattiva):
+
+`node`
+
+- Esegue il codice JavaScript che viene specificato come argomento:
+
+`node -e "{{codice}}"`
+
+- Mostra come risultato le versioni delle dipendenze di node:
```

Comment:
```suggestion
- Valuta un'espressione e stampa il risultato, utile per vedere le versioni delle dipendenze di node:
```

This translation is wrong. The original intent was to say that this can be used to return the result of an expression, the fact that it is being used to list the version of a package is coincidental.

Reply:
Yeah, you're right. I did not know the `-p` flag and I tried to interpret the original translation. My bad, I will fix it.